system-helper/org.freedesktop.Flatpak.rules.in

```diff
@@ -1,5 +1,6 @@
 polkit.addRule(function(action, subject) {
     if ((action.id == "org.freedesktop.Flatpak.app-install" ||
+         action.id == "org.freedesktop.Flatpak.configure-remote" ||
```

Comment:
Configure-remote was explicitly left out of this group because changing the configuration should require a password, whereas installing should be fine without it (for wheel users) because it was implicitly trusted by the gpg key that was added to the remote when the password was supplied with configure-remote.